Ginger/GingerCoreNET/Run/GingerRunner.cs

```diff
@@ -994,6 +994,11 @@ public void RunAction(Act act, bool checkIfActionAllowedToRun = true, bool moveT
             catch(Exception ex)
             {
                 Reporter.ToLog(eLogLevel.ERROR, "Exception in Run Action", ex);
+                if (string.IsNullOrEmpty(act.Error))
+                {
+                    act.Error = "Exception in Run Action " + ex;
```

Comment:
adding entire ex into Act.Error will add full stack strace to the error. And on Report it will break the layout. Please add only ex.message